### DISCUSSION TITLE
ggtb

### DISCUSSION THREAD

**Discussion by alshan** [JetBrains/lets-plot#1148] [Show and tell]
@Episkiliski, @OSuwaidi, @araichev, @maxkrivobok, @aeturrell
Hello folks, good news, you were selected for the first human trials of mouse drag and also wheel!
Access code: `p + ggtb()` 
Prerequisites: [v4.4.0](https://github.com/JetBrains/lets-plot?tab=readme-ov-file#what-is-new-in-440)
Terms of use: sharing your frustration is obligatory.
Cheers.

**Comment by araichev** [comment]
That's pretty cool.
Zooming with mouse scroll wheel does not work for me, though.
Is it supposed to?

I tested with v4.4.0 in a Jupyter notebook with the following code.
```
import lets_plot as lp


lp.LetsPlot.setup_html()
p = (
    lp.ggplot() 
    + lp.geom_livemap() 
    + lp.ggtb()
)
p.show()
```
By the way, it would be more user-friendly and [PEP8](https://pep8.org) compliant if the Lets-Plot examples in the docs were written without star imports and backslash linebreaks, like the code above.

**Reply by alshan** [reply]
Hi, livemap is not compatible with ggtb (the livemap component is special, it's interactive in a different way).
ggtb is intended for regular plots.

**Reply by araichev** [reply]
OK, does that mean that zooming in livemap with the mouse scroll wheel is still an open issue, or am i doing something wrong?

**Reply by alshan** [reply]
Mouse wheel zooming is disabled on livemap and currently there is no way to enable it even with new ggtb. This is the matter of further refinement I guess. An open issue in a sense.

**Comment by aeturrell** [comment]
I'm really sorry, I've just not had bandwidth to try this out yet. Are you still in the market for feedback?

**Reply by alshan** [reply]
Hi, certainly! Since last release we only added one feature: `double-click` to reset the viewport keeping the current tool active.

**Comment by alshan** [comment]
Folks! the panning, zooming released : [v4.5.0](https://github.com/JetBrains/lets-plot?tab=readme-ov-file#what-is-new-in-450)
